client.go

```diff
@@ -1118,17 +1118,30 @@ func LinkAddress(source string) LinkOption {
 // This option can be used multiple times.
 func LinkProperty(key, value string) LinkOption {
 	return func(l *link) error {
-		if key == "" {
-			return errorNew("link property key must not be empty")
-		}
-		if l.properties == nil {
-			l.properties = make(map[symbol]interface{})
-		}
-		l.properties[symbol(key)] = value
-		return nil
+		return l.setProperty(key, value)
+	}
+}
+
+// LinkPropertyInt64 sets an entry in the link properties map sent to the server.
+//
+// This option can be used multiple times.
+func LinkPropertyInt64(key string, value int64) LinkOption {
+	return func(l *link) error {
+		return l.setProperty(key, value)
 	}
 }
 
+func (l *link) setProperty(key string, value interface{}) error {
+	if key == "" {
+		return errorNew("link property key must not be empty")
+	}
+	if l.properties == nil {
+		l.properties = make(map[symbol]interface{})
+	}
+	l.properties[symbol(key)] = value
+	return nil
+}
```

Comment:
Would you mind implementing this helper as an unexported link option?
ie, `func linkProperty(key string, value interface{}) LinkOption`